src/wp-content/themes/twentytwentyfive/patterns/banner-intro-image.php

```diff
@@ -39,7 +39,7 @@
 			<div class="wp-block-buttons">
 				<!-- wp:button -->
 				<div class="wp-block-button">
-					<a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Learn More', 'Button text of intro section.', 'twentytwentyfive' ); ?></a>
+					<a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Learn More', 'Main CTA button text.', 'twentytwentyfive' ); ?></a>
```

Comment:
In my opinion, the updated text is incorrect. This is not a "Main CTA button text".
The reason why the old text included "Intro section" is because this specific pattern is designed to be an intro.

**I would like to hear more opinions** but I think the context should be removed.